template

```diff
@@ -0,0 +1,46 @@
+package protocol
+
+import "github.com/iden3/iden3comm"
+
+const (
+	// {Prefix}RequestMessageType defines {Prefix} request type of the communication protocol
+	{Prefix}RequestMessageType iden3comm.ProtocolMessage = iden3comm.Iden3Protocol + "PUT CONTENT"
+	// {Prefix}ResponseMessageType defines {Prefix} response type of the communication protocol
+	{Prefix}ResponseMessageType iden3comm.ProtocolMessage = iden3comm.Iden3Protocol + "PUT CONTENT"
+)
+
+// {Prefix}RequestMessage represent Iden3message for {Prefix} request
+type {Prefix}RequestMessage struct {
+	ID       string                    `json:"id"`
+	Typ      iden3comm.MediaType       `json:"typ,omitempty"`
+	Type     iden3comm.ProtocolMessage `json:"type"`
+	ThreadID string                    `json:"thid,omitempty"`
+
+	Body {Prefix}RequestMessageBody `json:"body,omitempty"`
+
+	From string `json:"from,omitempty"`
+	To   string `json:"to,omitempty"`
+}
+
```

Comment:
No need. Request messages won't always exist in the specific protocol. It will be not always a response message for a specific protocol message. The protocol is asynchronous by default.